int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -458,6 +463,17 @@ function edqSetGlobalIntuitiveConfiguration() {
 		},
 	];
 }
+function setCountryField() {
+	setTimeout(function() {
+		if (edqCountryLineId != null) {
+			if (edqCountryLineId.value === "") {
+				if (edqAddressLine1Id.hasAttribute("edq-metadata")) {
+					edqCountryLineId.value = countryAlpha2(window.EdqConfig.GLOBAL_INTUITIVE_ISO3_COUNTRY);
+				}
+			}
+		}
+	}, 5000);
```

Comment:
Why this particular value?

Reply:
Is a random value that I choose to make the timeout wait that I considered not too long for the timeout to be activated.